src/generators/prop_types/declarations.ts

```diff
@@ -38,7 +38,6 @@ export function buildTypeDeclarationFile(context: Context) {
               }),
             }),
           ],
-          declare: true,
```

Comment:
@ctumolosus is this OK to remove? I see you've only got VariableDeclarator's here, so I don't think `declare` does anything in that case

Reply:
Yes, I believe this change should be fine.